docs/self-managed/quickstart/developer-quickstart/c8run.md

```diff
@@ -229,51 +227,137 @@ To confirm that Camunda 8 Run has stopped, check for active containers:
 docker ps
 ```
 
-## Advanced options
+## Advanced configuration
 
 ### Enable TLS
 
-TLS can be enabled by providing a local keystore file using the `--keystore` argument at startup. Camunda 8 Run accepts `.jks` certificate files.
-Although C8Run supports TLS, this is intended only for testing.
+TLS can be enabled by providing a local keystore file using the `--keystore` argument at startup. Camunda 8 Run accepts `.jks` certificate files.  
+Although Camunda 8 Run supports TLS, this is intended only for testing.
 
 ### Access metrics
 
-Metrics are enabled in Camunda 8 Run by default and can be accessed at [http://localhost:9600/actuator/prometheus](http://localhost:9600/actuator/prometheus). For more information, see the [metrics](/self-managed/operational-guides/monitoring/metrics.md) documentation.
+Metrics are enabled in Camunda 8 Run by default and can be accessed at [http://localhost:9600/actuator/prometheus](http://localhost:9600/actuator/prometheus).  
+For more information, see the [metrics](/self-managed/operational-guides/monitoring/metrics.md) documentation.
+
+### Configure secondary storage (Elasticsearch or H2)
+
+Camunda 8 Run supports multiple secondary storage options.  
+By default, it uses **Elasticsearch**, but you can switch to **H2** for lightweight local development.
+
+In version **8.9-alpha1**, Camunda 8 Run starts with **Elasticsearch** as its default secondary storage.  
+To test Camunda 8 Run using an **H2 database**, configure H2 as an alternative secondary storage. This is useful for development and local testing because it requires minimal setup and runs entirely in memory.
+
+**Default (Elasticsearch) configuration:**
+
+```yaml
+data:
+  secondary-storage:
+    type: elasticsearch
+```
+
+**Optional (H2) configuration:**
```

Comment:
Thanks @christinaausley for adding it.

> data:
  secondary-storage:
    type: rdbms
    rdbms:
      url: jdbc:h2:mem:camunda
      username: sa
      password:
      flushInterval: PT0.5S
      queueSize: 1000

For alpha1, this config only does not work for h2. I like it how you separated it - but this also needs to set (else it will not start). Should we add this to the top (else it can be confusing as users who wants to try out will report errors due to the webapp being enabled + operate not starting):

>   backup:
    webapps:
      enabled: false

and disable operate:
> spring:
  profiles:
    active: "broker,consolidated-auth,identity,tasklist"